cinder/scripts/ceph-service.sh

```diff
@@ -22,6 +20,8 @@ do
 done
 
 
+# Create the pool
+ceph osd pool create images 64
```

Comment:
We can now use the stock ceph entrypoint, since this is the only important line we had to add there.